src/downloader.rs

```diff
@@ -43,7 +49,7 @@ impl Downloader {
         if sha256 != file.sha256 {
```

Comment:
We could maybe also print the downloaded size (since we already have a streaming wrapper that sees all the downloaded bytes, or we can just read the downloaded file size), just to have a double check in the CI log how large the file was.

Reply:
I'll add it in a followup PR.

Reply:
OK, since I had to change this PR anyway due to a build failure I ended up implementing this here :D